recipes/seqeval/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "seqeval" %}
+{% set version = "0.0.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9c7764b88b6d1430ab59de63efddf19495fca82159920fd11e362a405f8fc378
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py27]
```

Comment:
Why are you skipping `py27` here? According to the metadata in the `setup.py`, it supports it.